src/v/redpanda/admin/api-doc/features.json

```diff
@@ -177,6 +192,37 @@
                     "description": "Contents of a valid, signed license if loaded"
                 }
             }
+        },
+        "enterprise_feature": {
+            "id": "enterprise_feature",
+            "description": "One enterprise feature by name, and whether it is in use",
+            "properties": {
+                "name": {
+                    "type": "string"
+                },
+                "enabled": {
+                    "type": "boolean"
+                }
```

Comment:
non-blocking: if we want to include instructions on how to disable a particular feature, we should do that broker-side and then adding something like a string "disabling_instructions" field might make sense. (Not a blocker for this PR, we can add this later depending on product preferences.)
```suggestion
                },
                "disabling_instructions": {
                    "type": "string"
                }
```